presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -2314,6 +2314,22 @@ public void testNonEqualityJoinWithTryInFilter()
     }
 
     @Test
+    public void testTryPushedDownLeftJoinClause()
+            throws Exception
+    {
+        // this pushdowns TRY function to the sub filters
```

Comment:
Why would you say this is pushing stuff down? It looks like there are simply two identical expressions